README.md

```diff
@@ -1 +1,34 @@
-# Hello-world-Microverse
\ No newline at end of file
+### Prerequisites
+Test ID:M1d22022
+
+### Setup
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [https://github.com/Duncannjau])
+- Twitter: [https://www.linkedin.com/in/duncan-gitahi-07ba5851/)
+- LinkedIn: [https://twitter.com/Njaugitahi)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Guidance from Microverse team
+- Inspiration from Omar.
+
+
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] God job for updating README file But your READMEFile doesn't have description. README file should be descriptive and  all main headers from the [README template](https://github.com/microverseinc/readme-template/blob/master/README.md) should be present in your Repo.  Kindly, Update it.